tests/contrib_end_to_end_test.py

```diff
@@ -55,3 +56,35 @@ def test_mnist_contrib(self):
             big_dataset_score = current_df.loc[current_df.dataset_fraction_of_partner == 0.9, "contributivity_score"]
 
             assert small_dataset_score.values < big_dataset_score.values
+
+    def test_all_contrib_methods(self):
+        """
+        Test all available contributivity methods on mnist
+        """
+
+        all_methods = constants.CONTRIBUTIVITY_METHODS.copy()
+        all_methods.remove('AIS_Kriging_S')  # This one fails
+
```

Comment:
Why not removing IS reg S ?

Reply:
Why we should remove them ?

Reply:
You specified another test with it, and commented that this method would not start with less than 4 partners